### PR TITLE
Update file_collection_and_discovery.md

### PR DESCRIPTION
**Pull request by LLyaudet**
### Description 
typo "you your" -> "your"

### Checklist - did you ...

- [x] Add a CHANGELOG entry if necessary?
- [x] Add / update tests if necessary?
- [x] Add new / update outdated documentation?
Nothing was necessary :)

### CODE REVIEW

File: docs/usage_and_configuration/file_collection_and_discovery.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ as .pyi, and whether string normalization was omitted.
 
 To override the location of these files on all systems, set the environment variable
 `BLACK_CACHE_DIR` to the preferred location. Alternatively on macOS and Linux, set
-`XDG_CACHE_HOME` to you your preferred location. For example, if you want to put the
+`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the
 cache in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```
Comment (JelleZijlstra) [inline docs/usage_and_configuration/file_collection_and_discovery.md:28]:
```suggestion
`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the
cache in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```
Prettier

Comment (felix-hilden) [reply docs/usage_and_configuration/file_collection_and_discovery.md:28]:
```suggestion
`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the cache
in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```
I'm.. 99 % sure your suggestion is identical to the origin, did you mean the above?

Comment (felix-hilden) [reply docs/usage_and_configuration/file_collection_and_discovery.md:28]: Oh of course it shows the diff. So yeah, I think the `_Black_` on the next line won't fit the previous so this should satisfy Prettier.

**Review by ichard26 — APPROVED** [review]
Wonderful, thank you!

**Comment by ichard26** [comment]
Congrats on your first contribution here! :tada:

**Pull request merged**